doc/content/lfr/cardinal_7pincell/Cardinal_7pin_LFR_demo.md

```diff
@@ -0,0 +1,438 @@
+The transferred heat source is normalized to preserve the original total power using MOOSE’s conservative transfer features. 
+For transferring the temperature field from H.C. to Griffin for Doppler and coolant density feedback, temperature values at the three closest nodes in the H.C. mesh to the target node in the Griffin mesh are
+interpolated using inverse distance weighting (see +MultiAppGeometricInterpolationTransfer+ [!citep](Lindsay2022)).
+Nearest points are searched only in the solid region for solid temperatures and in the fluid region for fluid temperatures. 
+Even though H.C. does not need the fluid domain for solving the heat
+conduction equation, the H.C. problem contains a mesh for the fluid domain in order to receive the coolant temperature field transferred from Cardinal (on its way to Griffin) and compute its L2-norm for convergence check. 
+Note that Griffin does not need a fine mesh in the coolant region for
+neutronics solution, and a fine coolant mesh in H.C. does not degrade computational efficiency except in memory-limited cases.
+
+!media lfr/cardinal_7pin/transfer.png
+       style=width:100%
+       id=transfer
+       caption=Data transfer scheme among Griffin, H.C., and Cardinal.
+
+For the conjugate heat transfer, H.C. computes constant monomial heat flux per element at the cladding outer surfaces and the duct inner surface. 
+Note that the heat flux variable computed by H.C. is an elemental type, while the variable automatically added by Cardinal on the mirror mesh
+of the NekRS native mesh, is the nodal type. 
+Thus, the heat flux value at the nearest centroid of an element in the H.C. mesh is taken to the target node in the Cardinal mesh using
++MultiAppGeneralFieldNearestNodeTransfer+ [!citep](Lindsay2022). 
+If there are multiple nearest centroids near parallel process boundaries, parallel resolution between nearly equi-distant points may be inaccurate unless extense searches are done over every process. 
+To avoid this, the +greedy_search+ option was turned on to check the points of all the processors. This transfer takes place separately for different interfaces (e.g. duct-coolant and clad-coolant) with spatial restrictions on source and target boundaries for the search. 
+To preserve the total energy, the total power produced in the fuel rods
+and in the duct are separately transferred to Cardinal for normalization of the transferred heat flux at each of interfaces by Cardinal.
+
+The solid-fluid wall temperature is transferred from Cardinal to H.C. in the same manner as the transfer of heat flux. 
```

Comment:
In Nek, T is stored at the grid point level. Since Nek runs a much finer resolution, I believe an averaging or interpolation scheme is probably in place to process the Nek data and then transfer it to HT.

Reply:
The variable type for T in Cardinal is nodal 1st lagrange.

Reply:
on both the CFD and the HC side?

If so, the transfer can be quite simple if the meshes match.
If the meshes dont match I think we do not average or interpolate here, simply take the nearest value

If not, then same issue as the other direction

Reply:
Variable types are the same but the meshes are different.
Heat flux, surface temperature and bulk temperatures are all transferred using `MultiAppGeneralFieldNearestLocationTransfer`.